recipes/godot_headers/all/conanfile.py

```diff
@@ -0,0 +1,31 @@
+from conans import ConanFile
+from conans import tools
+
+
+class GodotHeadersConan(ConanFile):
+    name = "godot_headers"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/godotengine/godot_headers"
+    description = "Godot Native interface headers"
+    topics = ("game-engine", "game-development")
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        source_data = self.conan_data["sources"][self.version]
+        filename = source_data["url"].split("/")[-1]
+        commit = filename.split(".")[0]
+        tools.get(**source_data)
+        tools.rename("godot_headers-{}".format(commit), self._source_subfolder)
```

Comment:
It's ok, but it's simpler to use `glob`. Imho.
(+ `import glob` at top of the file)
```suggestion
        tools.get(**self.conan_data["sources"][self.version])
        tools.rename(glob.glob("godot_headers-*")[0], self._source_subfolder)
```